content/docs/reference/config/policy-and-telemetry/attribute-vocabulary/index.md

```diff
@@ -46,7 +46,7 @@ deployments will have agents (Envoy or Mixer adapters) that produce these attrib
 | `destination.service.uid`       | string | Unique identifier of the destination service. | `istio://istio-system/services/istio-telemetry` |
 | `destination.service.name`      | string | Destination service name. | `istio-telemetry` |
 | `destination.service.namespace` | string | Destination service namespace. | `istio-system` |
-| `request.headers` | map[string, string] | HTTP request headers. For gRPC, its metadata will be here. | |
+| `request.headers` | map[string, string] | HTTP request headers with lowercased keys. For gRPC, its metadata will be here. | |
```

Comment:
need to fix spelling error
s/lowercased/lowercase/ or add it to .spelling

Reply:
Done